### PR TITLE
Fixed issue #27095 - In Floating cart delete pop-up, When user click on out side of OK button than button color is changed

### PR DESCRIPTION
**Pull request by dipeshrangani**
### Related Pull Requests

https://github.com/magento/partners-magento2ee/pull/177

### Preconditions (*)

1. Magento2.4-develop

### Reference issue (*)
#27095 

### Steps to reproduce (*)

1. Add to cart product
2. Goto mini cart and click on remove product

### Expected result (*)

button-solution

### Actual result (*)

button-issue

### CODE REVIEW

**Comment by rogyar** [comment]
Hi @dipeshrangani. Thank you for your contribution. 

According to the [definition of done]()https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing_dod.html) all changes should be covered by automated tests. I would suggest covering your fix using MFTF test. The test will check that the following selector `action-primary action-accept` is not present on the page (but `action primary action-accept'`).

Thank you!

**Review by rogyar — CHANGES_REQUESTED** [review]
Hi @dipeshrangani. Thank you for your contribution. 

According to the [definition of done](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing_dod.html) all changes should be covered by automated tests. I would suggest covering your fix using MFTF test. The test will check that the following selector `action-primary action-accept` is not present on the page (but `action primary action-accept'`).

Thank you!

**Comment by dipeshrangani** [comment]
@rogyar I have updated requested changes, please review.

**Comment by rogyar** [comment]
Hi @dipeshrangani. I believe you don't need additional changes to your fix. As I wrote in my recent comment, you need to create an automated MFTF test that tests the new system behavior. You can check [this](https://github.com/magento/magento2/pull/27149) pull request as an example. It contains the fix itself and the automated test that verifies the fix.

**Comment by engcom-Echo** [comment]
I will take care of test coverage.

**Comment by engcom-Echo** [comment]
@magento run all tests

File: dev/tests/js/jasmine/tests/app/code/Magento/Ui/base/js/modal/confirm.test.js [hunk 1]
```diff
@@ -10,8 +10,21 @@ define([
...
+
+        beforeEach(function () {
+            element = $('<div id="element">some element</div>');
             confirm = element.confirm({});
+            widget = element.confirm({}).data('mage-confirm');
+        });
+
+        afterEach(function () {
+            $('element').remove();
+
```
Comment (rogyar) [inline dev/tests/js/jasmine/tests/app/code/Magento/Ui/base/js/modal/confirm.test.js]: Could we remove this extra empty line, please?

**Review by rogyar — CHANGES_REQUESTED** [review]

**Comment by rogyar** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
Failed functional tests not related to the changes in this PR

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-7241](https://jira.corp.magento.com/browse/ENGCOM-7241) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Comment by engcom-Echo** [comment]
@magenro run all tests

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by rogyar** [comment]
Hi @engcom-Echo. Thank you for the coverage. 

I believe the failing tests are not related to the current PR (they are fluky tests for Adobe Stock Integration).

**Comment by sivaschenko** [comment]
Adobe Stock Integration tests are stabilized. Updated branch to rerun the tests

**Comment by sivaschenko** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
Hi, @rogyar could you please approve it again?

**Review by rogyar — DISMISSED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-7241](https://jira.corp.magento.com/browse/ENGCOM-7241) has been created to process this Pull Request

**Review by VladimirZaets — CHANGES_REQUESTED** [review]
Hi @dipeshrangani. Looks like the issue mentioned in PR is already fixed. Can you please verify it?

**Comment by engcom-Echo** [comment]
Hi, @dipeshrangani, @VladimirZaets. I checked it again, and now this issue can't reproduce - an issue has been fixed. 

image

So, I'm closing this PR now due to an issue is fixed.
Please reopen and update if you wish to continue.
Thank you for your collaboration. :+1:

**Pull request closed without merging**